starter/source/tools/seatbelts/create_seatbelt.F

```diff
@@ -174,6 +176,9 @@ SUBROUTINE CREATE_SEATBELT(IXR,ITAB,KNOD2EL1D,NOD2EL1D,IPM,
         ENDIF
       ENDDO
 C
+C     Mat_seatbelt not used - nothing to do
+      IF ((NB_ELEM_1D==0).and.(NB_ELEM_2D==0)) RETURN       
```

Comment:
It's not recommended to have multiple RETURN in a routine

Can you change this please ?

Reply:
done